cmd/setup/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/AikidoSec/safechain-agent/internal/setup"
+	"github.com/AikidoSec/safechain-agent/internal/version"
+)
+
+func main() {
+	var showVersion = flag.Bool("version", false, "Show version information")
+	flag.Parse()
+
+	if *showVersion {
+		fmt.Print(version.Info())
+		os.Exit(0)
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
mm if we're always going to do it, cancel seems like the wrong name? what is this actually doing?